client/src/components/open-comment-panel-button/open-comment-panel-button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+// Helpers
+import classNames from 'classnames/bind'
+
+// Styles
+import styles from './open-comment-panel-button.scss'
+
+const css = classNames.bind(styles)
+
+const OpenCommentPanelButton = ({ hidden, onClick }) => (
```

Comment:
This should be merged with survey-panle-button and changed to a standard open-button component as they are the same except for the text